Package@swift-6.swift

```diff
@@ -798,6 +798,17 @@ extension Target {
       swiftSettings: [.swiftLanguageVersion(.v5)]
     )
   }
+
+  static var grpcHealth: Target {
+    .target(
+      name: "GRPCHealth",
+      dependencies: [
+        .grpcCore,
+        .grpcProtobuf
+      ],
+      path: "Sources/Services/Health"
```

Comment:
This will be changed later with the work George is doing, but for now we should probably add the `swiftSettings` param: `swiftSettings: [.swiftLanguageVersion(.v5)]`